### ISSUE TITLE
Unable to train data

### ISSUE THREAD

**Issue by Ash4059** [AbrahamSanders/seq2seq-chatbot#36]
Epoch:   1/500, Batch:  600/1731, Stats for last 100 batches: (Training Loss:  6.148, Training Time: 2301 seconds), Stats for epoch: (Training Loss:  6.632, Training Time: 12916 seconds)

Epoch:   1/500, Batch:  500/1731, Stats for last 100 batches: (Training Loss:  6.206, Training Time: 2229 seconds), Stats for epoch: (Training Loss:  6.729, Training Time: 10615 seconds)

Epoch:   1/500, Batch:  400/1731, Stats for last 100 batches: (Training Loss:  6.291, Training Time: 2176 seconds), Stats for epoch: (Training Loss:  6.860, Training Time: 8385 seconds)

**Comment by Ash4059** [comment]
I'm training data for the last about 4 hours It's working only on one epoch
